### DISCUSSION TITLE
How is the Fresh site documentation generated from Markdown?

### DISCUSSION THREAD

**Discussion by admercs** [freshframework/fresh#2309] [Q&A] [answered]
I am curious and must ask: how is the Fresh site documentation generated from Markdown? It is quite lovely.

Thank you!

**Comment by marvinhagemeister** [comment, marked as answer]
The code we use to render the markdown can be found here https://github.com/denoland/fresh/blob/d19c22d5921bdf21de9a7d7bc043b8523ac81893/www/utils/markdown.ts . We use a custom markdown renderer for `marked`. If you want a ready made library for that many folks use this one https://deno.land/x/gfm@0.6.0 .
